two minor issues (#3)
github-developer/example-actions-flux-eks: README.md

1. Create an EKS cluster, e.g. using [`eksctl create cluster`](https://eksctl.io/)
1. Set up Flux on the cluster, e.g. using [this guide](https://docs.fluxcd.io/en/latest/tutorials/get-started.html). Note that you must set `--git-path` to point to where your manifests are. For example:
```bash
export GHOWNER=<github user or organization account where your fork lives>

> export GHORG=<github org if repo is in an org, othewise github user again>

Reviewer: I think this line about `GHORG` can be deleted now
```suggestion
```

1. Set up Flux on the cluster, e.g. using [this guide](https://docs.fluxcd.io/en/latest/tutorials/get-started.html). Note that you must set `--git-path` to point to where your manifests are. For example:
```bash

> export GHUSER=<github user account where your fork lives>

Reviewer: Good thought, but could we instead rename `GHUSER` to `GHOWNER` to also include organizations?
```suggestion
export GHOWNER=<github user or organization account where your fork lives>
```

export GHORG=<github org if repo is in an org, othewise github user again>
export GHREPO=example-actions-flux-eks

kubectl create ns flux

fluxctl install \
    --git-user=${GHUSER} \
    --git-email=${GHUSER}@users.noreply.github.com \

>     --git-url=git@github.com:${GHORG}/${GHREPO} \

Reviewer: ```suggestion
    --git-url=git@github.com:${GHOWNER}/${GHREPO} \
```
